Tests/StreamChatTests/Workers/Background/MessageEditor_Tests.swift

```diff
@@ -93,7 +93,7 @@ final class MessageEditor_Tests: XCTestCase {
 
         // Check the state is eventually changed to `syncing`
         AssertAsync.willBeEqual(messageRepository.updatedMessageLocalState, .syncing)
-        XCTAssertCall("updateMessage(withID:localState:completion:)", on: messageRepository, times: 1)
+        XCTAssertCall("updateMessage(withID:localState:isBounced:completion:)", on: messageRepository, times: 1)
```

Comment:
@polqf As I mentioned previously, this is the reason why I don't think this test helper is very useful, although you might save some time when writing it, you waste more time whenever you need to update the test because the function name was changed or a new param was added. With a simple `updateMessageCallCount` this would be more simple and pragmatic IMO